quickstart.md

```diff
@@ -931,6 +931,9 @@ Confidential Containers integrates several components. If you run into problems,
 it can sometimes be difficult to figure out what is going on or how to move forward.
 Here are some tips.
 
+If you get stuck or find a bug, please make an issue on this repository or
```

Comment:
I would encourage users to open issues on confidential-containers/operator as this is the main entry point. Unless, of course, it is a problem on the documentation itself.

Reply:
Hm that's an interesting idea. We were gravitating towards this repo for issues because it has a lot of people following it, but maybe the operator makes more sense? wdyt @dcmiddle